src/mongocxx/exception/error_code.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 MongoDB Inc.
+// Copyright 2015-present MongoDB Inc.
```

Comment:
Good eye!

Reply:
Ok, I'll try to resolve these as I run into them! Thanks!